### ISSUE TITLE
Cannot import Interfaces in Typescript

### ISSUE THREAD

**Issue by code-snips** [newcat/baklavajs#411]
Following the example from the [docs](https://v2.baklava.tech/getting-started.html#creating-your-first-node),
typescript cant find these interfaces (it works with @ts-ignore):
```ts
import { SliderInterface, IntegerInterface, NumberInterface } from "baklavajs";
```
vscode suggests the import from a different path:
```ts
import { SliderInterface, IntegerInterface, NumberInterface } from "node_modules/@baklavajs/renderer-vue/dist";
```
i have installed baklavajs 2.4.4

is this issue related to the type override export similar to (https://github.com/newcat/baklavajs/discussions/396)?

**Comment by suhli** [comment]
same here, @baklavajs/renderer-vue type system seems broken

**Comment by newcat** [comment]
I released [v2.4.5](https://github.com/newcat/baklavajs/releases/tag/v2.4.5) that includes the fix from #409. Please check if that fixes your issue since I am currently unable to reproduce the problem in my projects.
